CONTRIBUTING.md

```diff
@@ -22,6 +22,9 @@ Instructions for contributing to [plaid-node][1]. A node.js client library for t
 ## Running Tests
 
 ```console
+SECRET=$PLAID_SECRET \
+CLIENT_ID=$PLAID_CLIENT_ID \ 
+PUBLIC_KEY=$PLAID_PUBLIC_KEY \
 $ make test
```

Comment:
Nit: the `$` in this example (representing the shell prompt) should really precede line 25.